consensus/service/src/byzantine_ledger/worker.rs

```diff
@@ -88,7 +88,8 @@ pub struct ByzantineLedgerWorker<
     network_state: SCPNetworkState,
 
     // Ledger sync service
-    ledger_sync_service: LedgerSyncService<L, PC, ReqwestTransactionsFetcher>,
+    // ledger_sync_service: LedgerSyncService<L, PC, ReqwestTransactionsFetcher>,
+    ledger_sync_service: Box<dyn LedgerSync<SCPNetworkState>>,
```

Comment:
This is the end goal: ledger_sync_service can now accept an instance of MockLedgerSync.